src/main/java/com/somemore/global/exception/ExceptionMessage.java

```diff
@@ -0,0 +1,15 @@
+package com.somemore.global.exception;
+
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@Getter
+public enum ExceptionMessage {
+
+    NOT_EXISTS_CENTER("존재하지 않는 기관 ID 입니다.")
+    ;
```

Comment:
```suggestion
    NOT_EXISTS_CENTER("존재하지 않는 기관 ID 입니다.")
```

;  <-- 이거 없애는걸 권장하는 것 같더라구요. 어떻게 생각하시나요?

Reply:
message라는 필드가 존재하는 열거형 클래스라 ;없이는 에러가 발생해서 넣어줬습니다
혹시 ;를 제거할 수 있는 방법이 있는건가요?